kafe2/fit/indexed/cost.py

```diff
@@ -1,96 +1,21 @@
-from .._base import CostFunctionBase, CostFunctionBase_Chi2, CostFunctionBase_NegLogLikelihood, \
```

Comment:
Same here. Those Classes to nothing except for a `pass` statement. This file can be removed.

Reply:
Keeping the cost functions for more intuitive use. As discussed previously.